Procfile

```diff
@@ -1,3 +1,3 @@
-stories: cd demo; bin/rails view_component_storybook:write_stories_json
-rails: cd demo; STORYBOOK=true bin/rails s -p 4000
+stories: cd demo; bundle install; bin/rails view_component_storybook:write_stories_json
```

Comment:
@manuelpuyol any idea how to avoid needing this `bundle install`?

Reply:
I'm not sure, I haven't run into the issues described tbh 🤔